src/bitcoin-chainstate.cpp

```diff
@@ -0,0 +1,263 @@
+// The bitcoin-chainstate executable serves to surface the dependencies required
```

Comment:
missing copyright header?

Reply:
Fixed as of: f0f78cb20b49f0a022bf3661538db17eb58ad280